src/libxively/io/fs/xi_fs_api.h

```diff
@@ -10,47 +10,8 @@
 #include <stdlib.h>
```

Comment:
haha I like the latter, you?  :)  I think so long as we update the files that we edit, then the coypright still holds for all files.